Dockerfile

```diff
@@ -66,8 +66,7 @@ RUN /bin/sh -c set -ex; \
 
 RUN /bin/sh -c set -ex; \
     dpkg-divert --divert /usr/bin/gcc.orig --rename /usr/bin/gcc; \
-    dpkg-divert --divert /usr/bin/g++.orig --rename /usr/bin/g++; \
-    update-alternatives --install /usr/bin/cc cc /usr/local/bin/gcc 999
+    dpkg-divert --divert /usr/bin/g++.orig --rename /usr/bin/g++;
```

Comment:
Why do you need this block ? ISTM that you could remove it completely if you don't want to mess with system compiler.

Reply:
I don't mind removing the entire block - My philosophy was leaving as many lines of the Dockerfile untouched as possible, but I'm fine with changing it more :)

Reply:
Fixed in a732373